src/text_engine/interaction/ConsolePrompt.java

```diff
@@ -51,18 +51,18 @@ public static int getResponseInt(BufferedReader reader, PrintWriter writer, Stri
      *
      * @param reader          to read response from
      * @param writer          to print prompt to
-     * @param defaultResponse the default response of the player, if the player submits empty input.
+     * @param prompt          to present to the player when asking for input
+     * @param defaultChoice the default response of the player, if the player submits empty input.
      *                        If {@code null}, an explicit response is required.
-     * @param prompt          to present to the user when asking for input
      * @return the player's response
      */
-    public static boolean getResponseBoolean(BufferedReader reader, PrintWriter writer,
-                                             Boolean defaultResponse, String prompt) {
-        boolean response;
+    public static boolean getChoiceBoolean(BufferedReader reader, PrintWriter writer,
+                                           String prompt, Boolean defaultChoice) {
+        boolean choice;
         while (true) {
-            Printers.printBooleanPrompt(writer, defaultResponse, prompt);
+            Printers.printBooleanPrompt(writer, prompt, defaultChoice);
             try {
-                response = getResponse(defaultResponse, reader.readLine());
+                choice = getChoice(defaultChoice, reader.readLine());
```

Comment:
"Choice" is shorter and allows for easy differentiation from the `String` that the user inputs and the `boolean` choice they make.